packages/flutter/lib/src/painting/continuous_rectangle_border.dart

```diff
@@ -26,27 +46,158 @@ import 'edge_insets.dart';
 ///
 /// See also:
 ///
-/// * [RoundedRectangleBorder] Which creates rectangles with rounded corners,
-///   however its straight sides change into a rounded corner with a circular
-///   radius in a step function instead of gradually like the
-///   [ContinuousRectangleBorder].
+/// * [RoundedRectangleBorder] which creates a rectangle whose corners are
+///   precisely quarter circles.
+/// * [ContinuousStadiumBorder] which creates a stadium whose two edges have a
+///   continuous transition into it's two 180º curves.
 class ContinuousRectangleBorder extends ShapeBorder {
-  /// The arguments must not be null.
+  /// Creates a Continuous Cornered Rectangle Border.
+  ///
+  /// The [side], [mode] and [borderRadius] arguments must not be null.
   const ContinuousRectangleBorder({
     this.side = BorderSide.none,
-    this.borderRadius = BorderRadius.zero,
+    this.borderRadius = 1.0,
   }) : assert(side != null),
        assert(borderRadius != null);
 
   /// The radius for each corner.
   ///
-  /// Negative radius values are clamped to 0.0 by [getInnerPath] and
-  /// [getOuterPath].
-  final BorderRadiusGeometry borderRadius;
+  /// The radius will be clamped to 1 if a value less than 1 is entered as the
+  /// radius.
+  ///
+  /// By default the radius is 1.0. This value must not be null.
+  ///
+  /// Unlike [RoundedRectangleBorder], there is only a single border radius used
+  /// to describe the radius for every corner.
+  final double borderRadius;
```

Comment:
we shouldn't call this borderRadius if it doesn't take a BorderRadius like other fields called borderRadius or accept elliptical radii. `circularRadiusValue` perhaps.

Reply:
How about corner radius?